sdk/servicebus/service-bus/karma.conf.js

```diff
@@ -49,9 +49,10 @@ module.exports = function(config) {
     // https://www.npmjs.com/package/karma-env-preprocessor
     envPreprocessor: [
       "SERVICEBUS_CONNECTION_STRING",
-      "AAD_CLIENT_ID",
-      "AAD_CLIENT_SECRET",
-      "AAD_TENANT_ID"
+      "SERVICEBUS_CONNECTION_STRING_BROWSER",
```

Comment:
We shouldnt be needing `SERVICEBUS_CONNECTION_STRING_BROWSER` anymore.
The arm template should be updating `SERVICEBUS_CONNECTION_STRING` separately for node and browser

Reply:
Hmmm, logged an issue too. https://github.com/Azure/azure-sdk-for-js/issues/7320
Will do this in this PR and close the issue too.